src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/SimpleOutputCollection.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKPath;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+/**
+ * To Use this class add a @ArgumentCollection variable like so:
+ *
+ * <code>
+ *
+ * @ArgumentCollection public final out = new SimpleOutput();
+ * </code>
+ * <p>
+ * and in the method <code>onTraversalSuccess</code> instead of just outputting to the terminal, also call
+ *
+ * <code>
+ * out.writeToOutput(value)
+ * </code>
+ * <p>
+ * where <code>value</code> is the object to be written to the file.
+ * <p>
+ * The code will only attempt to write to the output if -o (or --output) has been assigned on the commandline.
+ */
+public class SimpleOutputCollection implements Serializable {
```

Comment:
The name of this `ArgumentCollection` should clearly indicate both that the included arg is optional, and that it writes the Objects it's given as Strings. I suggest renaming to `OptionalTextOutputArgumentCollection`